ddtrace/internal/utils/version.py

```diff
@@ -47,3 +49,31 @@ def parse_version(version):
         parsed.release[1] if len(parsed.release) >= 2 else 0,
         parsed.release[2] if len(parsed.release) >= 3 else 0,
     )
+
+
+def _get_version_agent_format():
```

Comment:
```suggestion
def _pep440_to_semver(version=None):
    version = version or get_version()
```

Is maybe a bit more clear. The agent likely just supports semver so we can be a bit more general here.

Adding the argument makes the function more testable and reusable in case there are other instances we need to convert.